api/main.go

```diff
@@ -22,6 +32,48 @@ func main() {
 		baseGroup.GET("/countries/:country", endpoints.GetCountry)
 	}
 
-	address := fmt.Sprintf(":%d", PORT)
-	router.Run(address)
+	return router
+}
+
+func makeFrontendRouter() http.Handler {
+	router := gin.Default()
+	defaultCors := cors.Default()
+
+	// Mount middleware
+	router.Use(defaultCors)
+
+	router.StaticFS("/", gin.Dir(FRONTEND_DIR, true))
```

Comment:
not essential but worth taking a look at https://github.com/gin-contrib/static. Might be useful for symplifying what we have but may not.